src/server.h

```diff
@@ -579,6 +579,7 @@ typedef enum {
     PAUSE_BY_CLIENT_COMMAND = 0,
     PAUSE_DURING_SHUTDOWN,
     PAUSE_DURING_FAILOVER,
+    PAUSE_THROTTLE_CLIENT_WRITE,
```

Comment:
i think it should be renamed to `PAUSE_OOM_THROTTLE`
1. we can't say we throttle without mentioning what's the throttle. e.g. imagine in the future we'll have multiple throttles, each one needs to controlled separately.
2. the fact we throttle writes is a different argument to the pause function (we have a purpose and an action)